src/frameworks/qwik.json

```diff
@@ -9,7 +9,7 @@
   },
   "dev": {
     "command": "vite",
-    "port": 3000,
+    "port": 5173,
```

Comment:
For this one @ascorbic, it looks like Qwik has set the port to 5173 if you create a new Qwik app. Since this project is so new, I imagine for this config change it's OK?